snippets/list-entities-query-fields.mdx

```diff
@@ -6,7 +6,6 @@ export const listAccountsQueryFields = {
         {name:"invoice_currency",type: "string", description: "Currency in which the account is invoiced (`USD`, `EUR`, `INR`…)"}
     ],
     allowed_combinations: [
-        ["account_id"],
```

Comment:
Leave it be in the docs and please change the core-service code to accept account_id as a filter field.